### PR TITLE
Events calendar

### PR DESCRIPTION
**Pull request by GuillaumeOcculy**

### CODE REVIEW

**Comment by kophyo** [comment]
Hi there, please forgive me for the lack of information. I don't have any details of you applying for the job. Could you please let me know which channel did you use to apply? Thanks. Phyo

**Comment by GuillaumeOcculy** [comment]
Hi, I come from Aimee Backhouse - Nixor Resource Consulting

File: spec/features/creates_event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#     fill_in 'event_starts_at', with: Time.now
+#     fill_in 'event_ends_at', with: Time.now
+#
+#     fill_in 'event_description', with: "Guillaume"
+#
+#     click_button "Create Event"
+#
+#     expect(page).to have_content("Guillaume")
+#   end
+# end
```
Comment (kophyo) [inline spec/features/creates_event_spec.rb]: Any reason why this is being commented out?

Comment (GuillaumeOcculy) [inline spec/features/creates_event_spec.rb]: it was not working and I had no time to fix it

**Pull request closed without merging**